evervault-core/src/commonMain/kotlin/com/evervault/sdk/core/models/FunctionRunResponse.kt

```diff
@@ -0,0 +1,15 @@
+package com.evervault.sdk.core.models
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class FunctionResponseResult(
```

Comment:
I don't think this is accurate. Why do we want to limit the output of a function to only return a message?